File: Backend/src/main/java/cat/copernic/backendProjecte3/api/auth/dto/LoginRequest.java
package cat.copernic.backendProjecte3.api.auth.dto;

/**
 * Dades d'entrada per iniciar sessió.
 *
 * @param email Email de l'usuari.
 * @param password Contrasenya en text pla.
 */
public record LoginRequest(String email, String password) {

}

File: Mobile/app/src/main/java/cat/copernic/odecoches/data/remote/dto/LoginRequest.kt
package cat.copernic.odecoches.data.remote.dto

/**
 * Petició de login.
 *
 * @param email Correu electrònic de l'usuari.
 * @param password Contrasenya en text pla.
 */
/**
 * Petició amb les credencials necessàries per iniciar sessió.
 */
data class LoginRequest(
    val email: String,
    val password: String
)
